distribution/docs/src/main/resources/content/_configuring/platform-ui-theming.adoc

```diff
@@ -0,0 +1,31 @@
+:title: Configuring UI Theme
+:type: configuration
+:status: published
+:parent: Admin Theming Configurations
+:summary: Configure theme of all pages displayed on ${branding}.
+:order: 02
+
+== {title}
+
+The Platform UI Configuration allows for the customization of attributes of all pages within ${branding}.
+It contains settings to display messages to users at login or in banners in the headers and footers of all pages,
+along with changing the colors of text and backgrounds.
+
+=== Navigating to UI Theme Configuration
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-platform}* application.
+. Select *Configuration*.
+. Select *Platform UI Configuration*.
+
+=== Customizing the UI Theme
+
+The customization of the UI theme across ${branding} is available through the capabilities of Platform UI Configuration.
+There are four main areas that are able to be configured:
+
+. *Header*
+. *Footer*
+. *Text Color*
+. *Background Color*
```

Comment:
✏️ Since this was a point of confusion, should we clarify it's the background and text color for the banners?

Reply:
FIxed

Reply:
Maybe add `(text)` in parentheses after Header and Footer to make it clear what's being updated?